sys/include/net/gnrc/ipv6/nib/abr.h

```diff
@@ -45,7 +58,59 @@ int gnrc_ipv6_nib_abr_add(const ipv6_addr_t *addr);
  * @param[in] addr  The address of an authoritative border router.
  */
 void gnrc_ipv6_nib_abr_del(const ipv6_addr_t *addr);
-#endif  /* (GNRC_IPV6_NIB_CONF_6LBR && GNRC_IPV6_NIB_CONF_MULTIHOP_P6C) || defined(DOXYGEN) */
+#else   /* GNRC_IPV6_NIB_CONF_6LBR || defined(DOXYGEN) */
+#define gnrc_ipv6_nib_abr_add(addr)     (-ENOTSUP)
+#define gnrc_ipv6_nib_abr_del(addr)     (void)(addr)
```

Comment:
Found it!

Reply:
Ah... yeah. A non BR shouldn't be able to remove its BR. Otherwise it can lead to inconsistencies in the network ;-).